### DISCUSSION TITLE
Can set the RenderScaling with android?

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#12768] [Q&A] [answered]
```
TopLevel.RenderScaling
```
Can get the scaling, but it can't set, can set the `RenderScaling`?

**Comment by maxkatz6** [comment, marked as answer]
No. It doesn't make sense, as this value is reported by the system and you can't change it. 

If you want to scale content, use Viewbox.

**Reply by Coloryr** [reply]
ok, thanks.
